### ISSUE TITLE
Feature request: Recognize Kindle Fire devices

### ISSUE THREAD

**Issue by akarelas** [oalders/http-browserdetect#124]
I need to display a link to the correct appstore. But Amazon Kindle Fire devices have Amazon's appstore, instead of the standard Google Android one.

So knowing that a device is an Amazon Kindle Fire one would help.

Currently I have the following line to detect Fire devices. But this line needs to be maintained by someone. So I was hoping it would be incorporated in this module and maintained by you.

```perl
my $is_amazon = $ua->android && ($ua->silk or $ua_string =~ /\bKF[A-Z]+\s+Build\// or $ua_string =~ /\bkindle\b/i);
```

**Comment by andrewmoise** [comment]
I made a pull request for this - #125.

**Comment by andrewmoise** [comment]
Should be fixed as of version 3.11.
